airflow/providers/apache/druid/hooks/druid.py

```diff
@@ -89,7 +89,8 @@ def submit_indexing_job(self, json_index_spec: dict[str, Any] | str) -> None:
         self.log.info("Druid ingestion spec: %s", json_index_spec)
         req_index = requests.post(url, data=json_index_spec, headers=self.header, auth=self.get_auth())
         if req_index.status_code != 200:
-            raise AirflowException(f'Did not get 200 when submitting the Druid job to {url}')
+            raise AirflowException(f'The error code of {url}: {req_index.content}')
```

Comment:
I find the wording here a little misleading, you're not actually logging a satus code, but rather the response body which contains the error message and traces. Maybe something like:

```suggestion
            raise AirflowException(f'Failed when submitting the Druid job to {url}. Status code: {req_index.status_code}, response: {req_index.content}')
```

Reply:
Hi Nikolas,

Good advice. I changed that.